citesphere/src/main/java/edu/asu/diging/citesphere/web/user/ItemController.java

```diff
@@ -38,6 +38,7 @@ public String getItem(Authentication authentication, Model model, @PathVariable(
             @RequestParam(required = false, defaultValue = "", value = "conceptIds") String[] conceptIds) throws GroupDoesNotExistException, CannotFindCitationException, ZoteroHttpStatusException {
         ICitation citation = citationManager.getCitation((IUser)authentication.getPrincipal(), zoteroGroupId, itemId);
         model.addAttribute("zoteroGroupId", zoteroGroupId);
+        model.addAttribute("itemId", itemId);
```

Comment:
why is this needed? `itemId` is the id of the citation, isn't it? And the citation object is already in the model.

Reply:
Removed itemId.